canonical-web-and-design/react-components
# Modal accessibility improvements

## Done

- Add `aria-modal` to dialog
- Close on ESC key

## QA

### QA steps

- Open modal examples
  - https://react-components-445.demos.haus/?path=/story/modal--default-story
- Make sure modal can be closed using esc key


### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```
